artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverter.java

```diff
@@ -963,4 +966,13 @@ private static void setAMQMsgObjectProperties(final ActiveMQMessage amqMsg,
          }
       }
    }
+
+   private static byte[] getBytesPropertyOrNull(ICoreMessage coreMessage, SimpleString property) {
+      try {
+         return coreMessage.getBytesProperty(property);
```

Comment:
this shouldn't be catching property conversion exception, instead it should check if coremessage has property and return null, should still be allowing ActiveMQPropertyConversionException to propagate up if there's an actual conversion issue.